data/core/encyclopedia/geography.cfg

```diff
@@ -141,45 +175,60 @@ Over the River Aethen, south of Fort Tahn, is a Wesnothian frontier region. It i
+" +
+        #po: Geography of the Far North
+        _ "<header>text='Notable cities:'</header>
+               • Barag Gor: A city home to the Orcish Council.
+               • Bitok
+               • Borstep
+               • Farzi
+               • Jotha: A large fortified capital of the local merfolk that lies within the bay of Jotha.
+               • Lmarig
+               • Melmog
+               • Prestim
+               • Tirigaz: A coastal orcish town that occasionally raids Jotha.
+               • Dorest: The northernmost human city." + "
+" +
+        #po: Geography of the Far North
+        _ "<header>text='Notable land features:'</header>
+               • Desert of Death: An unsually dry desert in the Far North filled with venomous Sand Scuttlers. It is extremely hot by day yet chilly by night.
+               • Black Marshes: A perilous swamp north of Tirigaz filled with violent monsters.
+               • Mountains of Dorth: A set of mountains north of the Heart Mountains. Though nowhere near as dangeous as its southern neighbour, it is nonetheless deadly to inexperienced travelers.
+               • Mountains of Haag: A group of high mountains inhabited by groups of orcs, trolls and dwarves.
+               • Greenwood: An elvish forest located near the Lintanir Forest. Some say it was once connected to it, until groups of savage orcs burned much of the forest.
+               • Silent Forest: A dense forest inhabited by a few elves, once known as Gitamoth.
+               • Forest of Thelien: A dense coniferous forest sometimes inhabited by elves.
+               • River Oumph: A river flowing from the Mountains Haag to its delta just south of Bitok.
+               • River Bork: A river of the Far North flowing from a lake next to the Mourned Hills.
+               • Frosty Wastes: A cold, snowy desert in the far north inhabited only by the occasional monster.
+               • Barren Plains: A vast, sparsely populated snowy tundra."
+[/topic]
 
-<header>text='Notable land features:'</header>
-    • Black Marshes
-    • Mountains of Dorth
-    • Mountains of Haag
-    • Greenwood
-    • Silent Forest
-    • Forest of Thelien
-    • River Oumph
-    • River Bork
-    • Frosty Wastes
-    • Barren Plains"
+[topic]
+    id=far_south
+    title= _ "Far South"
+    text= _ "A vast region south of Kerlath home to the Dunefolk Civilization. It is inhabited by many local creatures, including the Jinni and Wyverns." + "
+
+" +
+        #po: Geography of the Far South
+        _ "<header>text='Notable land features:'</header>
+               • Sandy Wastes: A hot desert located south of Kerlath home to various Dunefolk settlements, as well as nomadic tribes inbetween. The northern parts of this desert are home to many treacherous monsters.
+               • Mountains of Peril: A tall and deadly mountain range in the far south inhabited by bands of orcish barbarians and other monsters.
```

Comment:
Orcs this far south? Instead of orcs could hint at another race, but not that picky.

Reply:
This is from this thread’s proposals, but could be removed if it does not fit in: https://forums.wesnoth.org/viewtopic.php?p=676020#p676020 The time in which this is set is not clear, and orcs could reasonably be here in my opinion given how far south orcs were during Asheviere’s reign (based on HttT).